ds/audioOutputPortConfig.cpp

```diff
@@ -108,8 +121,59 @@ List<AudioOutputPortType>  AudioOutputPortConfig::getSupportedTypes()
 	return supportedTypes;
 }
 
+void dumpconfig(audioConfigs_t *config)
+{
+	INT_INFO("%d:%s: Entering function\n", __LINE__, __func__);
+	INT_INFO("%d:%s: pKConfigs = %p\n", __LINE__, __func__, config->pKConfigs);
+	INT_INFO("%d:%s: pKPorts = %p\n", __LINE__, __func__, config->pKPorts);
+	INT_INFO("%d:%s: pKConfigSize %p = %d \n", __LINE__, __func__, config->pKConfigSize, *(config->pKConfigSize));
+	INT_INFO("%d:%s: pKPortSize %p = %d \n", __LINE__, __func__, config->pKPortSize, *(config->pKPortSize));
+
+	INT_INFO("\n\n=========================================================================================================================\n\n");
```

Comment:
i see many comment from yova regardig null check so go thought all the changes and make sure pointer has null check